prov/efa/test/efa_unit_test_ep.c

```diff
@@ -85,17 +85,17 @@ void test_rxr_ep_ignore_non_hex_host_id(struct efa_resource **state)
 void test_rxr_ep_handshake_exchange_host_id(struct efa_resource **state, uint64_t local_host_id, uint64_t peer_host_id, bool include_connid)
 {
 	fi_addr_t peer_addr = 0;
-	int ret;
+	int cq_read_recv_ret, cq_read_send_ret;
 	struct efa_ep_addr raw_addr = {0};
 	size_t raw_addr_len = sizeof(raw_addr);
 	struct efa_rdm_peer *peer;
 	struct efa_resource *resource = *state;
-	struct efa_unit_test_buff recv_buff;
 	struct efa_unit_test_handshake_pkt_attr pkt_attr = {0};
 	struct fi_cq_data_entry cq_entry;
 	struct ibv_qp *ibv_qp;
 	struct rxr_ep *rxr_ep;
 	struct rxr_pkt_entry *pkt_entry;
+	uint64_t actual_peer_host_id = peer_host_id + 1;
```

Comment:
Why does the " + 1"?

Reply:
> Why does the " + 1"?

This ensures `actual_peer_host_id != peer_host_id` at the beginning of the test.

Reply:
I think use `(uint64_t)-1` is better

Reply:
Cool. Incorporated in update. Thanks!